itemframework/src/item/abstract_item.cpp

```diff
@@ -325,6 +327,38 @@ ItemOutput* AbstractItem::addOutput(int type, QString const& description)
     return output;
 }
 
+void AbstractItem::removeInput(ItemInput* itemInput)
```

Comment:
This could be expressed using `std::find` (or `qFind`) which would also make the weird `return` redundant.